exchanges/coinbene/coinbene_websocket.go

```diff
@@ -254,14 +258,24 @@ func (c *Coinbene) wsHandleData(respRaw []byte) error {
 			})
 		}
 		for j := range orderBook.Data[0].Bids {
-			amount, err = strconv.ParseFloat(orderBook.Data[0].Bids[j][1], 64)
+			price, err = strconv.ParseFloat(orderBook.Data[0].Bids[j][0], 64)
 			if err != nil {
 				return err
 			}
-			price, err = strconv.ParseFloat(orderBook.Data[0].Bids[j][0], 64)
+
+			if price == 0 {
+				// Last level is coming back as a float with not enough decimal
+				// places e.g. ["0.000","1001.95"]],
+				// This needs to be filtered out as this can skew orderbook
+				// transversals
```

Comment:
What do you mean by "orderbook transversals"?

Reply:
traversal**

Reply:
I still don't understand though. Isn't it because it makes the orderbook inaccurate as they're not providing accurate enough data?

Reply:
Yes, when I say order book traversals or traversing an order book, I am meaning ranging across price levels and calculating data. I will change the comment if this is too confusing.